app/uk/gov/hmrc/bindingtariffclassification/model/reporting/ReportResult.scala

```diff
@@ -16,11 +16,14 @@
 
 package uk.gov.hmrc.bindingtariffclassification.model.reporting
 
+import uk.gov.hmrc.bindingtariffclassification.model.reporting.CaseReportGroup.CaseReportGroup
+
 case class ReportResult
 (
-  group: Option[String],
+  group: Map[CaseReportGroup, Option[String]],
```

Comment:
Note this will break the frontend, if you can think of a better way to do it than using a Map go for it